src/Themes/Theme.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Carnival\Themes;
+
+interface Theme {
+    
+    function getName() : string;
+
+    function getDescription() : string;
+
+    function getLogo() : string;
```

Comment:
I think we should leave out logos for now until we figure out how we want to handle images.
At the very least, it isn't obvious exactly what `getLogo` is returning in this state